src/useObserver.ts

```diff
@@ -7,24 +7,24 @@ import { useForceUpdate, useUnmount } from "./utils"
 export type ForceUpdateHook = () => () => void
 
 export interface IUseObserverOptions {
-    customForceUpdateHook?: ForceUpdateHook
+    useForceUpdate?: ForceUpdateHook
 }
 
 export function useObserver<T>(
     fn: () => T,
-    baseComponentName?: string,
+    baseComponentName: string = "observed",
     options?: IUseObserverOptions
 ): T {
     if (isUsingStaticRendering()) {
         return fn()
     }
 
-    const wantedForceUpdateHook = (options && options.customForceUpdateHook) || useForceUpdate
+    const wantedForceUpdateHook = (options && options.useForceUpdate) || useForceUpdate
     const forceUpdate = wantedForceUpdateHook()
```

Comment:
With default you can do this in here and nothing else.
```ts
const forceUpdate = options.useForceUpdate()
```

Reply:
not quite, if that's done and a new property is ever added to options and options does not contain useForceUpdate then that would break

e..g
```
useObserver(..., {
  someOtherOption: ...,
  // no useForceUpdate since it is optional
}

const forceUpdate = options.useForceUpdate() // useForceUpdate will be undefined 
```

Reply:
That's why I was suggesting default in previous comment :)